grpc/service_config/service_config.proto

```diff
@@ -233,6 +258,62 @@ message XdsConfig {
   google.protobuf.StringValue lrs_load_reporting_server_name = 5;
 }
 
+// Configuration for eds LB policy.
+message EdsLoadBalancingPolicyConfig {
+  // Cluster name.  Required.
+  string cluster = 1;
+
+  // EDS service name, as returned in CDS.
+  // May be unset if not specified in CDS.
+  string eds_service_name = 2;
+
+  // Server to send load reports to.
+  // If unset, no load reporting is done.
+  // If set to empty string, load reporting will be sent to the same
+  // server as we are getting xds data from.
+  google.protobuf.StringValue lrs_load_reporting_server_name = 3;
+
+  // Locality-picking policy.
+  // This policy's config is expected to be in the format used
+  // by the weighted_target policy.  Note that the config should include
+  // an empty value for the "targets" field; that empty value will be
+  // replaced by one that is dynamically generated based on the EDS data.
+  // Optional; defaults to "weighted_target".
+  repeated LoadBalancingConfig locality_picking_policy = 4;
```

Comment:
How can we make a default "weighted_target" config without any data?

Reply:
This is covered in the design doc.  Basically, the CDS policy will generate a config for the EDS policy that looks like this:

```
      "localityPickingPolicy": [
        // Note: The "targets" field in this config will be
        // overwritten by the eds_experimental policy, so it can
        // be empty here.
        {"weighted_target_experimental": {
          "targets": {}
        }}
      ],
```